engine/src/main/java/org/terasology/rendering/cameras/PerspectiveCamera.java

```diff
@@ -71,7 +72,7 @@ public boolean isBobbingAllowed() {
     @Override
     public void loadProjectionMatrix() {
         glMatrixMode(GL_PROJECTION);
-        GL11.glLoadMatrix(MatrixUtils.matrixToFloatBuffer(getProjectionMatrix()));
+        GL11.glLoadMatrix(getProjectionMatrix().get(BufferUtils.createFloatBuffer(16)));
```

Comment:
If this is something that gets called every frame, I'm itching to optimize the memory access here too, but that doesn't have to be in this PR.